qualysapi/contrib.py

```diff
@@ -37,7 +40,7 @@ def generate_vm_report(self, report_details, startup_delay=60, polling_delay=30,
     logger.debug('report_id: %s' % (report_id))
     # Wait for report to finish spooling.
     # Maximum number of times to check for report.  About 10 minutes.
-    MAX_CHECKS = 10
+    # MAX_CHECKS = 10
```

Comment:
Variables are case sensitive in Python (see below).  __max_checks__ is a function parameter (lines 27 and 46) while __MAX_CHECKS__ is created on line 43 but then never used again so it can safely be removed.

$ python3
```
>>> max_checks = True
>>> MAX_CHECKS = False
>>> print(max_checks, MAX_CHECKS)
True False
```